batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/db/DbUtilities.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.db;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * Created by leapfrog on 1/18/16.
+ */
+public class DbUtilities {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)